src/map/script_map.cpp

```diff
@@ -166,9 +177,22 @@ static int CclCenterMap(lua_State *l)
 }
 
 /**
+** <b>Description</b>
+**
 **  Define the starting viewpoint for a given player.
 **
 **  @param l  Lua state.
+**
+** Example:
+**
+** <div class="example">
+** 	<code>
+**		  -- Start view for player 0.<br>
+**		  SetStartView(0, 25, 12)<br>
+**		  -- Start view for player 1.<br>
+**		  SetStartView(0, 71, 38)
```

Comment:
I think this should be `SetStartView(1, 71, 38)` :)

Reply:
https://github.com/Wargus/stratagus/commit/72f4bce9a18b577169f05e468e99e81cce150a7d#r52423732